adapt-and-extend/theming/resourceregistry.rst

```diff
@@ -18,47 +18,72 @@ Introduction to Plone 5 resources
 
 Plone 5 introduces new concepts for working with JavaScript and CSS in Plone.
 
-We finally have a solution to define JavaScript modules and their dependencies - in contrast to the previous approach, where all dependencies had to be loaded in order.
-We achieve that with the tool RequireJS, which follows the Asynchronous Module Definition (AMD) approach.
-We chose AMD over other module loading implementations (like CommonJS) because AMD can be used in non-compiled form in the browser and also compiled Through-The-Web.
-This way we can also support a "development mode" like in Plone versions prior to 5, where changes in the JavaScript sources are instantly reflected (actually they are made available a few seconds after initial browser load).
+JavaScript and AMD
+------------------
+
+Prior to Plone 5, JavaScript resources in Plone have been registered with the ``portal_javascripts`` tool using the Generic Setup import step ``jsregistry.xml``.
+This approach allowed a developer to specify the order in which individual JavaScript resources were loaded into an HTML page, and controlled compiling and minifying these resources for production into a minimal set of files.
+As the use of JavaScript in front-end development expanded, this model proved insufficient to solve complex dependency management issues.
+At the same time, developments in the JavaScript community led to a new approach to handling complex dependencies.
+
+Starting in Plone 5, we use a new `module <http://requirejs.org/docs/why.html>`_-based solution on the `Asynchronous Module Definition <http://requirejs.org/docs/whyamd.html>`_ (AMD) approach.
+The new Plone 5 Resource Registry uses `RequireJS <http://requirejs.org/>`_ to allow developers to define individual JavaScript modules and the dependencies they will require.
+Because RequireJS uses the AMD approach, these modules and their dependencies can be served in an uncompiled form during development and then compiled and minified Through-The-Web for use in production.
+This allows a "development mode" where changes in JavaScript source files are reflected in the browser in near real-time.
+
+It is also possible in the new Resource Registry to provide bundles with simple Javascript that does not make use of AMD.
+See :ref:`bundle_example_non_amd` below for an example of such a bundle.
```

Comment:
@ebrehault this paragraph is added to address [your comment above](https://github.com/plone/documentation/pull/518/files#r52873699).